controller/src/main/java/io/pravega/controller/store/stream/StreamMetadataStore.java

```diff
@@ -1125,6 +1126,19 @@
     CompletableFuture<Map<UUID, ActiveTxnRecord>> getActiveTxns(final String scope, final String stream,
                                                                 final OperationContext context, final Executor executor);
 
+    /**
+     * Method to retrieve all active transactions from metadata store in completed state.
+     *
+     * @param scope Scope of stream
+     * @param stream name of stream
+     * @param limit max no of record
+     * @param continuationToken continuation token
+     * @param context operation context
+     * @param executor caller executor
+     * @return Pair of list of txIds with status and token
+     */
+    CompletableFuture<Pair<List<Controller.TxnResponse>, String>> listCompletedTxns(final String scope, final String stream, final int limit, final String continuationToken, final OperationContext context, final Executor executor);
```

Comment:
I see that the return type is same but the javadoc is not consistent

Reply:
Updated the javadoc.